### ISSUE TITLE
Cannot start ActiveMQ service via Homebrew on macOS 11.6.4 Big Sur

### ISSUE THREAD

**Issue by beechovsky** [Homebrew/homebrew-core#96511]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```
$ brew config
HOMEBREW_VERSION: 3.4.1
ORIGIN: https://github.com/Homebrew/brew
HEAD: ea0ddc3297739b7380887d4a7e6b6590c314dca9
Last commit: 28 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 6e3f2ec6dfc767a0c4855571cc8191fad02ca90f
Core tap last commit: 2 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_CORE_GIT_REMOTE: https://github.com/Homebrew/homebrew-core
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.jJ8ewfKp0z/org.xquartz:0
HOMEBREW_MAKE_JOBS: 16
Homebrew Ruby: 2.6.8 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.8/bin/ruby
CPU: 16-core 64-bit kabylake
Clang: 13.0.0 build 1300
Git: 2.32.0 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 11.6.4-x86_64
CLT: 13.2.0.0.1.1638488800
Xcode: 13.2.1
```
```
$ brew doctor
Your system is ready to brew.
```

### Verification

- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

I'm trying to run ActiveMQ on my MacBook running macOS 11.6.4 Big Sur (our apps communicate via JMS).

### Background

ActiveMQ installed and run via Homebrew is not working on Big Sur. This is the case for both an M1 machine and an Intel on the same OS over the past couple weeks.

Other services installed and run by Homebrew (Redis and MySQL) are behaving nominally.

### Diagnostics
NOTE: I've run brew update before, after, and in between each of these steps a few times to no avail.

UPDATE: It appears to be an issue with brew services, not ActiveMQ. 

Manually starting activemq works as expected:
```
$ /usr/local/opt/activemq/bin/activemq start
INFO: Loading '/usr/local/Cellar/activemq/5.16.4/libexec//bin/env'
INFO: Using java '/Library/Java/JavaVirtualMachines/jdk1.8.0_321.jdk/Contents/Home/bin/java'
INFO: Starting - inspect logfiles specified in logging.properties and log4j.properties to get details
INFO: pidfile created : '/usr/local/Cellar/activemq/5.16.4/libexec//data/activemq.pid' (pid '13725')
```

Finally, when I run `brew services start activemq --verbose`, I see:
```
...
/bin/launchctl enable gui/1591/homebrew.mxcl.activemq
/bin/launchctl bootstrap gui/1591 /Users/jbucklew/Library/LaunchAgents/homebrew.mxcl.activemq.plist
Bootstrap failed: 5: Input/output error
Error: Failure while executing; `/bin/launchctl bootstrap gui/1591 /Users/jbucklew/Library/LaunchAgents/homebrew.mxcl.activemq.plist` exited with 5.
```

```
$ brew services list
Name     Status  User     File
activemq stopped me ~/Library/LaunchAgents/homebrew.mxcl.activemq.plist
mysql    started me ~/Library/LaunchAgents/homebrew.mxcl.mysql.plist
redis    started me ~/Library/LaunchAgents/homebrew.mxcl.redis.plist
```

**Restarting to be sure:**
```
$ brew services restart activemq
Stopping `activemq`... (might take a while)
==> Successfully stopped `activemq` (label: homebrew.mxcl.activemq)
==> Successfully started `activemq` (label: homebrew.mxcl.activemq)
$ brew services list
Name     Status  User     File
activemq stopped me ~/Library/LaunchAgents/homebrew.mxcl.activemq.plist
mysql    started me ~/Library/LaunchAgents/homebrew.mxcl.mysql.plist
redis    started me ~/Library/LaunchAgents/homebrew.mxcl.redis.plist
```

`brew doctor` initially indicated I need to upgrade my version of XCode CLI tools, which I did. This is verified by the output of `brew doctor` and `brew config` above.

After a reboot after updating XCode CLI tools:
```
$ brew doctor
Your system is ready to brew.
```

Rechecking ActiveMQ after upgrading XCode CLI Tools shows the issue persists:
```
$ brew services list
Name     Status  User     File
activemq stopped me ~/Library/LaunchAgents/homebrew.mxcl.activemq.plist
mysql    started me ~/Library/LaunchAgents/homebrew.mxcl.mysql.plist
redis    started me ~/Library/LaunchAgents/homebrew.mxcl.redis.plist
$ brew services restart activemq
Stopping `activemq`... (might take a while)
==> Successfully stopped `activemq` (label: homebrew.mxcl.activemq)
==> Successfully started `activemq` (label: homebrew.mxcl.activemq)
$ brew services list
Name     Status  User     File
activemq stopped me ~/Library/LaunchAgents/homebrew.mxcl.activemq.plist
mysql    started me ~/Library/LaunchAgents/homebrew.mxcl.mysql.plist
redis    started me ~/Library/LaunchAgents/homebrew.mxcl.redis.plist
```

Uninstalling and reinstalling makes no difference.

Installing the official binary from the Apache ActiveMQ website works fine, so it probably isn't an issue with ActiveMQ.

### What did you expect to happen?

I expected the service to run nominally.

### Step-by-step reproduction instructions (by running `brew` commands)

Update brew.
```
$ brew update
Already up-to-date.
```

Install ActiveMQ.
```
$ brew install activemq
==> Downloading https://ghcr.io/v2/homebrew/core/activemq/manifests/5.16.4
Already downloaded: /Users/ME/Library/Caches/Homebrew/downloads/bce25bb83dcf0e2a6cec685518dbbffbdbf1f72eb46119d2c529c0607ed97480--activemq-5.16.4.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/activemq/blobs/sha256:5e729e282ec91d20565892e5af21e41901df77717bc4c38e4a0a9277e6443eeb
Already downloaded: /Users/ME/Library/Caches/Homebrew/downloads/354be0cc83c3f9b57e18b3e427c06d831f0d072221431e437d4c125ecf6dfda9--activemq--5.16.4.big_sur.bottle.tar.gz
==> Pouring activemq--5.16.4.big_sur.bottle.tar.gz
==> Caveats
To restart activemq after an upgrade:
  brew services restart activemq
Or, if you don't want/need a background service you can just run:
  /usr/local/opt/activemq/bin/activemq start
==> Summary
🍺  /usr/local/Cellar/activemq/5.16.4: 533 files, 70.8MB
==> Running `brew cleanup activemq`...
Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).
```

Check the service.
```
$ brew services list
Name     Status  User     File
activemq stopped me ~/Library/LaunchAgents/homebrew.mxcl.activemq.plist
mysql    started me ~/Library/LaunchAgents/homebrew.mxcl.mysql.plist
redis    started me ~/Library/LaunchAgents/homebrew.mxcl.redis.plist
```

Restart the service.
```
$ brew services restart activemq
Stopping `activemq`... (might take a while)
==> Successfully stopped `activemq` (label: homebrew.mxcl.activemq)
==> Successfully started `activemq` (label: homebrew.mxcl.activemq)
```

Check the service again.
```
$ brew services list
Name     Status  User     File
activemq stopped me ~/Library/LaunchAgents/homebrew.mxcl.activemq.plist
mysql    started me ~/Library/LaunchAgents/homebrew.mxcl.mysql.plist
redis    started me ~/Library/LaunchAgents/homebrew.mxcl.redis.plist
```

**Comment by SMillerDev** [comment]
What happens when you run `/usr/local/opt/activemq/bin/activemq start`?

Additionally, are you sure that's an M1 laptop? `CPU: 16-core 64-bit kabylake` suggests it isn't

**Comment by SMillerDev** [comment]
Okay, so it's not really ActiveMQ that's failing then. It's the way brew calls ActiveMQ in services.

What does `brew services info activemq` say?

**Comment by beechovsky** [comment]
Right. I've updated the above information to reflect that.

`$ brew services info activemq
activemq (homebrew.mxcl.activemq)
Running: ✘
Loaded: ✔
Schedulable: ✘
`

**Comment by SMillerDev** [comment]
Does `brew services run activemq` have the same problem?

**Comment by beechovsky** [comment]
Yup.

`$ brew services run activemq
Bootstrap failed: 5: Input/output error
Error: Failure while executing; `/bin/launchctl bootstrap gui/1591 /usr/local/opt/activemq/homebrew.mxcl.activemq.plist` exited with 5.`

**Comment by SMillerDev** [comment]
If ActiveMQ is working for anyone, could you post their `brew config`?

**Comment by cho-m** [comment]
I think `activemq start` kicks off daemon process. Could try running foreground `activemq console`

**Comment by cho-m** [comment]
I think `activemq start` kicks off daemon process. Could try running foreground `activemq console`

**Comment by cho-m** [comment]
Should be fixed by #96965 based on my quick tests, but can reopen if there are still issues.
